src/test/ui/async-await/issue-70935-complex-spans.stderr

```diff
@@ -0,0 +1,37 @@
+error: future cannot be sent between threads safely
+  --> $DIR/issue-70935-complex-spans.rs:10:45
+   |
+LL |   fn foo(tx: std::sync::mpsc::Sender<i32>) -> impl Future + Send {
+   |                                               ^^^^^^^^^^^^^^^^^^ future created by async block is not `Send`
+LL |
+LL | /     async move {
+LL | |         baz(|| async{
+LL | |             foo(tx.clone());
+LL | |         }).await;
+LL | |     }
+   | |_____- this returned value is of type `impl std::future::Future`
+   |
+   = help: the trait `std::marker::Sync` is not implemented for `std::sync::mpsc::Sender<i32>`
+note: future is not `Send` as this value is used across an await
+  --> $DIR/issue-70935-complex-spans.rs:13:9
+   |
+LL | /         baz(|| async{
+LL | |             foo(tx.clone());
+LL | |         }).await;
+   | |________________^
+note: first, await occurs here, with the value maybe used later
+  --> $DIR/issue-70935-complex-spans.rs:13:9
+   |
+LL | /         baz(|| async{
+LL | |             foo(tx.clone());
+LL | |         }).await;
+   | |________________^
+note: ...but, the value is later dropped here
```

Comment:
I don't think `...but` is clearer here. To me, it implies that the fact that it is dropped is part of the problem. And in the case of the borrow check errors that sound like this, it is.

Sometimes that's true in that you could drop the value earlier, before the await point. But in both of these cases it's not; the extra note is just information. (And not very useful information in these cases, but that's something we can clean up later.)

Anyway, it might be helpful to represent that this is a continuation of the above note, in which case I'd ask that we keep only the ellipsis, e.g. `note: ...the value is later dropped here`